aviary/utils/doctape.py

```diff
@@ -489,3 +494,105 @@ def get_function_names(file_path) -> set:
     ]
 
     return set(function_names)
+
+
+def glue_actions(cmd, curr_glued=[], glue_default=False, glue_choices=False, md_code=True):
+    """
+    Glue all Aviary CLI options
+
+    Parameters
+    ----------
+    cmd: str
+        Aviary command
+    curr_glued: list
+        the parameters that have been glued
+    glue_default: boolean
+        flag whether the default values should be glued.
+    """
+    parser = argparse.ArgumentParser()
```

Comment:
generally, you don't want to use a list as a default value for a function in python, so the preferred method would be like this
```suggestion
def glue_actions(cmd, curr_glued=None, glue_default=False, glue_choices=False, md_code=True):
    """
    Glue all Aviary CLI options

    Parameters
    ----------
    cmd: str
        Aviary command
    curr_glued: list
        the parameters that have been glued
    glue_default: boolean
        flag whether the default values should be glued.
    """
    if curr_glued is None:
        curr_glued = []
    parser = argparse.ArgumentParser()
```

Reply:
Done.